go.mod

```diff
@@ -149,7 +149,7 @@ require (
 	github.com/pquerna/otp v1.4.0
 	github.com/prometheus/client_golang v1.18.0
 	github.com/prometheus/client_model v0.5.0
-	github.com/prometheus/common v0.46.0
+	github.com/prometheus/common v0.45.0
```

Comment:
This matches prometheus/client_golang v1.18.0: https://github.com/prometheus/client_golang/blob/v1.18.0/go.mod#L10-L12